rosdep/python.yaml

```diff
@@ -7821,6 +7825,10 @@ python3-opencv:
     '7': null
     '8': null
   ubuntu: [python3-opencv]
+python3-opencv-python-headless-pip:
+  '*':
+    pip:
+      packages: [opencv-python-headless]
```

Comment:
Please remove this key. OpenCV Python libraries are already available from system packages https://index.ros.org/d/python-opencv/